java-api-examples/setup.sh

```
@@ -0,0 +1,29 @@
+    -DBUILD_SHARED_LIBS=ON \
+    -DSHERPA_ONNX_ENABLE_PORTAUDIO=OFF \
+    -DSHERPA_ONNX_ENABLE_JNI=ON \
+    ..
+
+  make -j4
+  ls -lh lib
+  popd
+fi
+
+# Build sherpa-onnx JVM jar if not exists
+if [ ! -f ../sherpa-onnx/java-api/target/sherpa-onnx-jvm-*.jar ]; then
```

Shellcheck (0.11.0)

[error] -f doesn't work with globs. Use a for loop.

(SC2144)